docs/pages/repo/docs/support.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Support Policy
+description: Our Support Policy for Turborepo and its ecosystem
+---
+
+## Support Policy
+
+`turbo` is generally supported on Debian-based Linux distributions, macOS, and
+Windows on both x86_64 and ARM 64 architectures. Specifically, we build and ship
+the following binaries via `npm`:
+
+- turbo-darwin-64 (macOS with Intel chips)
+- turbo-darwin-arm64 (macOS with Apple Silicon)
+- turbo-linux-64
+- turbo-linux-arm64
+- turbo-windows-64
+- turbo-windows-arm64
```

Comment:
We don't actually build arm64 binaries for windows right now, although we will once we're done with Go. Windows on arm ships with an emulation layer to run x86_64 binaries, so we ship the same binary as x86_64 windows and just call it arm

Reply:
i'll leave this juicy morsel out of the docs, since this reflects our aspirational "support" anyway

Reply:
not sure we need to list the binaries at all, but maybe will change in a future pass